doc/herbstluftwm.txt

```diff
@@ -478,12 +478,14 @@ bring 'WINID'::
 
 
 resize 'DIRECTION' ['FRACTIONDELTA']::
-    Changes the next fraction in specified 'DIRECTION' by 'FRACTIONDELTA' (which
-    defaults to 0.02 if none is supplied). 'DIRECTION' behaves as specified at
-    the 'focus' command. If a floating window is focused, it is resized and
-    grows into the specified 'DIRECTION'. In the parameter 'FRACTIONDELTA', you
-    should not omit the sign '-' or '+', because in future versions, the
-    behaviour may change if the sign is omitted. Example:
+    Changes the size of the focused frame (or floating window) in specified
+    'DIRECTION' by 'FRACTIONDELTA' (which defaults to 0.02 if none is
+    supplied). 'DIRECTION' behaves as specified at the 'focus' command. If a
+    floating window is focused, it grows towards the monitor edge of the
+    specified 'DIRECTION' ('FRACTIONDELTA' is ignored in that case). In the
+    parameter 'FRACTIONDELTA', you should not omit the sign '-' or '+',
+    because in future versions, the behaviour may change if the sign is
+    omitted. Example:
```

Comment:
The first sentence is much clearer now. :+1: 

The last sentence confused me a bit because I missed the "not" when first reading. How about fool-proofing it a bit like this? `An explicit '-' or '+' is recommended for the parameter 'FRACTIONDELTA', because in future versions, the behavior without a sign may change.`

But maybe first of all: Do we know what you had in mind here, @t-wissmann? Maybe we can drop that entire statement. It's been there for 9 years now.

Reply:
Yes we can drop it, because I'd never change the behaviour now. I think I had something in mind like setting the fraction of a frame split to a fixed value (e.g. resetting it to 0.5). I think this is what is requested in #99.

More importantly: `resize` command for floating windows does not only grow to the monitor edge, but actually to the next visible edge on the monitor, that is, to the edge of another window or to the monitor edge if there is no other window in between.

Reply:
I dropped the -+ sign text and updated the edge text